### ISSUE TITLE
Include audio file contributions under "Contributions" in the Profile Page "Stats"

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#1612]
Note that we have 3 or more contributors that have contributed audio files, but the "Contributions" are listed as 0.

Here's one example.

https://tatoeba.org/eng/user/profile/aleteacher2

<img width="381" alt="screen shot 2018-07-19 at 18 57 10" src="https://user-images.githubusercontent.com/7934273/42936065-20fb52fe-8b86-11e8-80ba-d1b19eee5824.png">

**Comment by jiru** [comment]
Yes, audio recordings are not marked as contributions. I agree that it’s quite confusing to audio recordings and zero contribution. Currently, the meaning of the number of "contributions" here only include whatever is logged in sentences history: creation, edition, deletion, linking, unlinking. It’s the number of items you can see in https://tatoeba.org/contributions/of_user/aleteacher2.

**Comment by ckjpn** [comment]
In that case, perhaps the easiest fix would just be to rename "Contributions" to "Logged Items".
This would eliminate the confusion and wouldn't look like a bug.

**Comment by alanfgh** [comment]
I don't think "Logged Items" would mean much to most people.

One alternative possibility would be "Other contributions", which would
have to go directly under "Audio recordings". However, that wording seems a
little dismissive with regard to sentence contributions. Another
possibility would be "Nonaudio contributions", but that has a similar
problem. Or we could use "Sentence actions".

**Comment by ckjpn** [comment]
"Sentence Actions" seems strange, too.  

Just brainstorming. Maybe one of these will trigger other ideas.

Sentence contributions, editing, deletions and linking
Text contributions, editing, deletions and linking
Contributions other than audio files

**Comment by ckjpn** [comment]
Perhaps consider the idea of dropping the "Contributions" number and replacing it with the following.  This would make it very clear and also perhaps be interesting.

Links Created
Sentences Deleted
Sentences Edited

**Comment by Poulpisator** [comment]
CK's suggestion would definitely make stats lover (like me) happy.    
Now that we have a system that separates original sentences from translations, we could even think of splitting "Sentences owned" into "Original sentences" and "Translations"

**Comment by jiru** [comment]
For the record, this issue has been raised during a [UX test](https://en.wiki.tatoeba.org/articles/show/ux-test-1) too:
> In the Stats block, M. is confused by the number of Contributions because it doesn’t add up with the other numbers.
